lib/community.js

```diff
@@ -79,6 +79,13 @@ exports.all = {
         quote: 'We are a Travel and Leisure company and we are using hapi.js for a few years in our projects for both Backend and mobile app APIs. hapi is fast, flexible and powerful!',
         person: 'Mehdi Karimian, Senior Engineer'
     },
+    'car2go': {
```

Comment:
Please just capitalize the name of the company

Reply:
That's actually how it's written — so capitalizing would violate our CI :D

Reply:
oh ok, I see nevermind then. 😄